commands/src/main/java/org/wildfly/extras/creaper/commands/transactions/ChangeBasicTransactionAttributes.java

```diff
@@ -0,0 +1,251 @@
+package org.wildfly.extras.creaper.commands.transactions;
+
+
```

Comment:
Just curious, why do you always have 2 empty lines between `package ...` and the first `import ...`? Looks weird.